.github/workflows/cross_docker.yaml

```
@@ -35,3 +38,29 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+  publish-trusted-sync:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    env:
+      IMAGE_NAME: ${{ github.repository }}/trusted-sync
+    steps:
+      - uses actions/checkout@v4
+      - uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
```

Think this needs a `password` field
```suggestion
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}
```